<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemaSyncDialog</class>
 <widget class="QDialog" name="SchemaSyncDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Синхронизация структуры БД</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../s11main.qrc">
    <normaloff>:/new/prefix1/icons/Icon2.png</normaloff>:/new/prefix1/icons/Icon2.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="grpServer">
     <property name="title">
      <string>Настройка эталонного сервера</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblSeverName">
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtServerName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lvlServerPort">
        <property name="text">
         <string>Порт</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="edtServerPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblDatabaseName">
        <property name="text">
         <string>База</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtDatabaseName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblUserName">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edtUserName"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblPassword">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="edtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>207</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnSync">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Синхронизация</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnClose">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../s11main.qrc"/>
 </resources>
 <connections/>
</ui>
